### DISCUSSION TITLE
Simulation of result of example e3_simple_project_bps_energyplus

### DISCUSSION THREAD

**Discussion by Xingjian-Wang** [BIM2SIM/bim2sim#478] [Q&A] [answered]
Hi there,

I ran the file e3_simple_project_bps_energyplus and I noticed that the results show all the energy consumption as 0, but I can see other information about the building, please see the attached screenshot. I would like to ask how can I simulate the energy consumption correctly?
I have a further question as well, how can I see the parameters used in the file about the building (e.g. building component materials, internal heat gain, etc.)?

Many thanks in advance
![e3 result energy comsumption](https://user-images.githubusercontent.com/72249299/218417893-399674fa-3973-46ac-bc80-7091026164ac.jpg)

Best Regards,
Xingjian Wang

**Comment by veronikarichter** [comment, marked as answer]
Hi Xingjiang, 

the default value for the simulation runtime is "design days only" by default. You can easily change it to an annual simulation by adding "project.workflow.run_full_simulation = True" to your example setup. 
You could also just take the resulting IDF-file, modify the simulation runtime in the file, and run it again directly in EnergyPlus. 
In this resulting IDF-file you can also have a look at the settings for the other parameters, such as materials and internal loads (and easily modify if needed).

I hope this helps!

**Reply by Xingjian-Wang** [reply]
Hi Veronika,

many thanks for your answer, this problem has been solved.

Regards,
Xingjian
